nilearn/datasets/atlas.py

```diff
@@ -963,3 +968,150 @@ def fetch_atlas_surf_destrieux(data_dir=None, url=None,
+    then gyrus, tissue, and ba).
+
+    The labels json file contains
+    [['level name', ['labels', 'for', 'this', 'level' ...]], ...],
+    where the levels are in the order mentionned above.
+
+    """
+    data_dir = _get_dataset_dir(
+        'talairach_atlas', data_dir=data_dir, verbose=verbose)
+    img_file = os.path.join(data_dir, 'talairach.nii')
+    labels_file = os.path.join(data_dir, 'talairach_labels.json')
+    if os.path.isfile(img_file) and os.path.isfile(labels_file):
+        return img_file, labels_file
+    atlas_url = 'http://www.talairach.org/talairach.nii'
+    temp_dir = mkdtemp()
+    try:
+        temp_file = _fetch_files(
+            temp_dir, [('talairach.nii', atlas_url, {})], verbose=verbose)[0]
+        atlas_img = nb.load(temp_file, mmap=False)
+        atlas_img = check_niimg(atlas_img)
+    finally:
+        shutil.rmtree(temp_dir)
+    labels = atlas_img.header.extensions[0].get_content()
+    labels = labels.strip().decode('utf-8').split('\n')
+    labels = [l.split('.') for l in labels]
+    new_img, level_labels = _separate_talairach_levels(
+        atlas_img, labels, verbose=verbose)
+    new_img.to_filename(img_file)
+    with open(labels_file, 'w') as fp:
+        json.dump(level_labels, fp)
+    return img_file, labels_file
+
+
+def fetch_atlas_talairach(level_name, data_dir=None, verbose=1):
+    """Download the Talairach atlas.
+
+    .. versionadded:: 0.3.2
+
+    Parameters
+    ----------
+    level_name : {'hemisphere', 'lobe', 'gyrus', 'tissue', 'ba'}
+        Which level of the atlas to use: the hemisphere, the lobe, the gyrus,
+          the tissue type or the Brodmann area.
+
+    data_dir : str, optional (default=None)
+        Path of the data directory. Used to force data storage in a specified
+        location.
+
+    verbose : int
+        verbosity level (0 means no message).
+
+    Returns
+    -------
+    sklearn.datasets.base.Bunch
+        Dictionary-like object, contains:
+
+        - maps: 3D Nifti image, values are indices in the list of labels.
+        - labels: list of strings. Starts with 'Background'.
+        - description: a short description of the atlas and some references.
+
+    References
+    ----------
+    http://talairach.org/about.html#Labels
+
+    `Lancaster JL, Woldorff MG, Parsons LM, Liotti M, Freitas CS, Rainey L,
+    Kochunov PV, Nickerson D, Mikiten SA, Fox PT, "Automated Talairach Atlas
+    labels for functional brain mapping". Human Brain Mapping 10:120-131,
+    2000.`
+
+    `Lancaster JL, Rainey LH, Summerlin JL, Freitas CS, Fox PT, Evans AC, Toga
+    AW, Mazziotta JC. Automated labeling of the human brain: A preliminary
+    report on the development and evaluation of a forward-transform method. Hum
+    Brain Mapp 5, 238-242, 1997.`
+    """
+    levels = _TALAIRACH_LEVELS
+    if level_name not in levels:
+        raise ValueError('"level_name" should be one of {}'.format(levels))
+    position = levels.index(level_name)
```

Comment:
I would prefer if you did not define the local variable "levels", but rather used the module global "_TALAIRACH_LEVELS". It helps following the code.